packages/sources/coinmetrics/src/transport/price-ws.ts

```diff
@@ -91,15 +91,16 @@ export const handleAssetMetricsMessage = (
     logger.error(message, `Error response from websocket`)
```

Comment:
While you're making changes here, could you include the error message in this log? Would be useful for future debugging of similar issues

```suggestion
    logger.error(message, `Error response from websocket: ${JSON.stringify(message)}`)
```